index.css

```diff
@@ -0,0 +1,16 @@
+body,
+h1 {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
You're giving the `body` and the `h1` the same styling? Is this intended?

Reply:
I guess I was in a hurry of some sort. Separated those 🥇